src/main/scala/loamstream/db/slick/ExecutionDaoOps.scala

```diff
@@ -143,9 +144,9 @@ trait ExecutionDaoOps extends LoamDao { self: CommonDaoOps with OutputDaoOps =>
   
   private def insertableExecutions(executions: Iterable[Execution]): Iterable[(Execution, CommandResult)] = {
     executions.collect {
-      case e @ Execution( _, _, _, Some(cr: CommandResult), _, _, _) => e -> cr
+      case e @ Execution( _, _, _, Some(cr: CommandResult), _, _, _, _) => e -> cr
```

Comment:
There probably isn't, but in Erlang, OCaml, and a couple other languages, it's possible to match by field name for records (case classes) instead of having to ignore every last field. For example, is this possible in Scala?

```scala
case class Point(x: Int, y: Int)

Point(1, 2) match {
  case p @ Point(y: y) => p -> y
}
```

I expect not, but if it is, it might be better to switch to that?

Reply:
There isn't a way to extract just one field like that; I've heard it mentioned though, maybe it's a new Dotty feature.  It's possible to define a custom extractor, such that you could say:
```scala
executions.collect {
  case e @ Execution.WithCommandResult(cr) => e -> cr
}
```
Or something.  I'll see about doing that.  These pattern matches have always felt a bit unseemly, since they drop some `Executions` without `CommandResult`s or command-lines on the floor, which may explain why I didn't try to make this tidy earlier.  That said, `Execution`s from things other than command-line jobs are an old concept that no longer makes sense in LS, so perhaps the best way forward is to assume `Executions` are the result of a command.  

At the very least, I'll try to neaten this up with a custom extractor.